src/main/resources/edu/cornell/kfs/coa/cu-spring-coa.xml

```diff
@@ -204,5 +205,30 @@
             </set>
         </property>
     </bean>
+    
+    <bean id="createClosedLaborAccountsCsvByPreviousDaysJob" parent="unscheduledJobDescriptor">
```

Comment:
How about the job renamed to createClosedLaborAccountsCsvJob

Reply:
Renamed.